app/js/actions/settingEncounterType.js

```diff
@@ -2,9 +2,9 @@ import axios from 'axios';
 import {
   SETTING_ENCOUNTER_TYPE_SUCCESS,
   SETTING_ENCOUNTER_TYPE_FAILURE,
-  SETTING_ENCOUNTER_TYPE_LOADING,
```

Comment:
Removing this line would also require removing the actionType from the test file.

Reply:
Thanks for your feedback. 
Based on the implementation test by @Betty-Kebenei, the loading action type is needed by other test blocks in the `settingEncounterType.test.js` file. 
This PR takes care of the `fetchConfigurationsLoading` created in `settingEncounter.js` file which was the reason for removing it.